src/effects/effectmanifest.h

```diff
@@ -93,6 +94,14 @@ class EffectManifest final {
         m_isMixingEQ = value;
     }
 
+    const bool& isWaveformChangingEQ() const {
```

Comment:
a bool should be retuned by value.

Reply:
> There is also an issue of a jumping sound changes when you use both HPF and LPF knob.

I can confirm that. Yesterday I've put some more time into testing this feature and noticed a slight tickle when turning on the megaknob to change both HPF and LPF. Turning on one of those singular knobs doesn't have that tickle ...